src/Maestro/DependencyUpdateErrorProcessor/.config/settings.Development.json

```diff
@@ -0,0 +1,8 @@
+{
+  "KeyVaultUri": "https://maestrolocal.vault.azure.net/",
+  "BuildAssetRegistry": {
+    "ConnectionString": "Data Source=localhost\\SQLEXPRESS;Initial Catalog=BuildAssetRegistry;Integrated Security=true"
+  },
+  "AppConfigurationUri": "https://maestrolocal.azconfig.io/",
+  "GithubUrl": "https://github.com/maestro-auth-test/maestro-test2"
```

Comment:
This could use a more descriptive name so that it's clear that this is the repo where we'll be opening issues at.